jung-visualization/src/test/java/edu/uci/ics/jung/visualization/spatial/rtree/RTreeTest.java

```diff
@@ -227,25 +229,15 @@ private void testAreas(TreeNode rootNode) {
     Rectangle2D rootBounds = rootNode.getBounds();
     if (rootNode instanceof InnerNode) {
       InnerNode innerNode = (InnerNode) rootNode;
-      Assert.assertTrue(closeEnough(rootBounds, Node.union(innerNode.getChildren())));
+      Rectangle2D unionBounds = Node.union(innerNode.getChildren());
+      assertThat(rootBounds.getMinX()).isWithin(1.0E-3).of(unionBounds.getMinX());
+      assertThat(rootBounds.getMinY()).isWithin(1.0E-3).of(unionBounds.getMinY());
+      assertThat(rootBounds.getMaxX()).isWithin(1.0E-3).of(unionBounds.getMaxX());
+      assertThat(rootBounds.getMaxY()).isWithin(1.0E-3).of(unionBounds.getMaxY());
     }
```

Comment:
using minX, maxX, minY, maxY instead of width and height, because Rectangle2D can have negative width/height. I care only that the overall dimensions are close enough that there is no gross error.